modules/odc-creating-functions.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-getting-started.adoc
+
+:_content-type: PROCEDURE
+[id="odc-creating-functions_{context}"]
+= Creating a function in the web console
+
+You can create a function from a Git repository by using the *Developer* perspective of the {product-title} web console.
+
+.Prerequisites
+
+* The {ServerlessOperatorName} and Knative Serving are installed on the cluster.
+* The {pipelines-shortname} Operator is installed on the cluster.
+* You have logged in to the {product-title} web console.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
+* You have created or have access to a Git repository that contains the code for your function. The repostory must contain a `func.yaml` file and use the `s2i` build strategy.
+
+[IMPORTANT]
+====
+Before you can create a function by using the web console, a cluster administrator must create the following tasks and pipeline, so that they are available for all namespaces on your cluster.
+
+* Create the `func-s2i` task:
++
+[source,terminal]
+----
+$ oc apply -f https://raw.githubusercontent.com/openshift-knative/kn-plugin-func/serverless-1.30/pkg/pipelines/resources/tekton/task/func-s2i/0.1/func-s2i.yaml
+----
+
+* Create the `func-deploy` task:
++
+[source,terminal]
+----
+$ oc apply -f https://raw.githubusercontent.com/openshift-knative/kn-plugin-func/serverless-1.30/pkg/pipelines/resources/tekton/task/func-s2i/0.1/func-s2i.yaml
+----
+
+* Create the Node.js function pipeline:
++
+[source,terminal]
+----
+$ oc apply -f https://raw.githubusercontent.com/openshift-knative/kn-plugin-func/serverless-1.30/pkg/pipelines/resources/tekton/task/func-s2i/0.1/func-s2i.yaml
+----
+====
```

Comment:
If the first two prerequisite items have to be completed by a cluster administrator, then I suggest considering joining them all into a single prerequisite item with a list of second-level items, and then adding the terminal commands as tips to separate second-level items. I mean something like this:

```suggestion

* A cluster administrator has completed the following steps:

** The {ServerlessOperatorName} and Knative Serving are installed on the cluster.

** The {pipelines-shortname} Operator is installed on the cluster.

** The `func-s2i` task is created for all namespaces on the cluster.
+
[TIP]
====
The administrator can run the following command to create the `func-s2i` task:
[source,terminal]
----
$ oc apply -f https://raw.githubusercontent.com/openshift-knative/kn-plugin-func/serverless-1.30/pkg/pipelines/resources/tekton/task/func-s2i/0.1/func-s2i.yaml
----
====

** The `func-deploy` task is created for all namespaces on the cluster.
+
[TIP]
====
The administrator can run the following command to create the `func-deploy` task:
[source,terminal]
----
$ oc apply -f https://raw.githubusercontent.com/openshift-knative/kn-plugin-func/serverless-1.30/pkg/pipelines/resources/tekton/task/func-s2i/0.1/func-s2i.yaml
----
====

** The Node.js function pipeline is created for all namespaces on the cluster.
+
[TIP]
====
The administrator can run the following command to create the Node.js function pipeline:
[source,terminal]
----
$ oc apply -f https://raw.githubusercontent.com/openshift-knative/kn-plugin-func/serverless-1.30/pkg/pipelines/resources/tekton/task/func-s2i/0.1/func-s2i.yaml
----
====
```

Reply:
I don't think "tips" are appropriate here, since these are the exact commands that need to be used by the cluster admins. I've made this just one step in the prereqs though instead of a note.